shared/src/main/kotlin/com/egm/stellio/shared/util/ApiResponses.kt

```diff
@@ -1,3 +1,5 @@
 package com.egm.stellio.shared.util
 
 const val ENTITY_NOT_FOUND_MESSAGE = "Entity Not Found"
+
+fun entityNotFoundMessage(entityId: String) = "Entity $entityId was not found"
```

Comment:
don't be selfish, migrate the existing uses of `ENTITY_NOT_FOUND_MESSAGE` (and avoid a duplicated message)

Reply:
OK ;-)